README.md

```diff
@@ -0,0 +1,188 @@
+| `400`     | Error JSON object | The wishlist `:Id` does not exist or not given |
+| `500`     | Error JSON object | Server error |
+
+### `PUT /wishlists/:Id`
+
+#### Description
+Updates a specific wishlist.
+
+#### Parameters
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlists JSON object | The wishlist name has been updated successfully |
+| `400`     | Error JSON object | Wishlist `:id` was not found or was missing |
+| `500`     | Error JSON object | Server error |
+
+### `DELETE /wishlists/:Id`
+
+#### Description
+Deletes a wishlist.
+
+#### Parameters
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlists JSON object | The JSON object of the wishlist deleted |
+| `400`     | Error JSON object | Wishlist `:id` was not found or was missing |
+| `500`     | Error JSON object | Server error |
+
+### `POST /wishlists/:Id/items/:ItemId`
+
+#### Description
+Adds an item to a wishlist
+
+#### Parameters
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `201`     | Wishlist item JSON object | The JSON object of the wishlist item added |
+| `400`     | Error JSON object | `:WishlistId` or `:ItemId` were not found or were missing |
+| `500`     | Error JSON object | Server error |
+
+### `GET /wishlists/:Id/items/:ItemId`
+
+#### Description
+Retrieves an item on a wishlist
+
+#### Parameters
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlist item JSON object | The JSON object of the wishlist item requests |
+| `400`     | Error JSON object | `:WishlistId` or `:ItemId` were not found or were missing |
+| `500`     | Error JSON object | Server error |
+
+### `PUT /wishlists/:Id/items/:ItemId`
+
+#### Description
+Updates information about an existing item on a wishlist
+
+#### Parameters
+None
+
+#### Body
+JSON key-value store of new information for the item.  Example:
+```
+{
+   ...
+}
+```
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlist item JSON object | The JSON object of the wishlist item with updated data |
+| `400`     | Error JSON object | `:WishlistId` or `:ItemId` were not found or were missing |
+| `500`     | Error JSON object | Server error |
+
+### `DELETE /wishlists/:Id/items/:ItemId`
+
+#### Description
+Deletes an item from a wishlist.
+
+#### Parameters
+None
+
+#### Body
+None
+
+#### Returns
+| HTTP code | Body | Description | 
+| --------- | ---- | ----------- |
+| `200`     | Wishlist item JSON object | The JSON object of the wishlist item deleted |
+| `400`     | Error JSON object | `:WishlistId` or `:ItemId` were not found or were missing |
+| `500`     | Error JSON object | Server error |
```

Comment:
I think we also need an endpoint for listing all Items in one wishlist? [Link](https://github.com/NYU-DevOps-Squad-Wishlists/wishlists/issues/10) @biegel

Reply:
this should be `GET /wishlists/:Id` above